api/app/signals/apps/api/v1/views/signal.py

```diff
@@ -52,7 +52,7 @@ class PrivateSignalViewSet(DatapuntViewSet,
     pagination_class = HALPagination
 
     authentication_classes = (JWTAuthBackend,)
-    permission_classes = (SIAPermissions,)
+    permission_classes = (SignalCreateInitialPermission,)
```

Comment:
Not entirely convinced of the naming here (almost implies a user must be authorized to create a signal to view the list endpoint).